android/cli/commands/_build.js

```diff
@@ -2719,209 +2781,116 @@ AndroidBuilder.prototype.copyResources = function copyResources(next) {
-			delete this.lastBuildFiles[bootstrapJsonAbsolutePath];
-
-			if (!jsFilesToEncrypt.length) {
-				// nothing to encrypt, continue
-				return next();
-			}
-
-			// figure out which titanium prep to run
-			let titaniumPrep = 'titanium_prep';
-			if (process.platform === 'darwin') {
-				titaniumPrep += '.macos';
-			} else if (process.platform === 'win32') {
-				titaniumPrep += '.win32.exe';
-			} else if (process.platform === 'linux') {
-				titaniumPrep += '.linux' + (process.arch === 'x64' ? '64' : '32');
-			}
-
-			// encrypt the javascript
-			const titaniumPrepHook = this.cli.createHook('build.android.titaniumprep', this, function (exe, args, opts, done) {
-				this.logger.info(__('Encrypting JavaScript files: %s', (exe + ' "' + args.slice(1).join('" "') + '"').cyan));
-				appc.subprocess.run(exe, args, opts, function (code, out, err) {
-					if (code) {
-						return done({
-							code: code,
-							msg: err.trim()
-						});
-					}
 
-					// write the encrypted JS bytes to the generated Java file
-					fs.writeFileSync(
-						path.join(this.buildGenAppIdDir, 'AssetCryptImpl.java'),
-						ejs.render(fs.readFileSync(path.join(this.templatesDir, 'AssetCryptImpl.java')).toString(), {
-							appid: this.appid,
-							encryptedAssets: out
-						})
-					);
-
-					done();
-				}.bind(this));
-			});
-			let args = [ this.tiapp.guid, this.appid, this.buildAssetsDir ].concat(jsFilesToEncrypt);
-			if (process.platform === 'win32') {
-				const fileListing = path.join(this.buildDir, 'titanium_prep_listing.txt');
-				args = [ this.tiapp.guid, this.appid, this.buildAssetsDir, '--file-listing', fileListing ];
-				fs.writeFileSync(fileListing, jsFilesToEncrypt.join('\n'));
-			}
-
-			const opts = {
-					env: appc.util.mix({}, process.env, {
-						// we force the JAVA_HOME so that titaniumprep doesn't complain
-						JAVA_HOME: this.jdkInfo.home
-					})
-				},
-				fatal = function fatal(err) {
-					this.logger.error(__('Failed to encrypt JavaScript files'));
-					err.msg.split('\n').forEach(this.logger.error);
-					this.logger.log();
-					process.exit(1);
-				}.bind(this);
-
-			titaniumPrepHook(
-				path.join(this.platformPath, titaniumPrep),
-				args.slice(0),
-				opts,
-				function (err) {
-					if (!err) {
-						return next();
-					}
-
-					if (process.platform !== 'win32' || !/jvm\.dll/i.test(err.msg)) {
-						fatal(err);
-					}
+				if (process.platform !== 'win32' || !/jvm\.dll/i.test(err.msg)) {
+					fatal(err);
+				}
 
-					// windows 64-bit failed, try again using 32-bit
-					this.logger.debug(__('32-bit titanium prep failed, trying again using 64-bit'));
-					titaniumPrep = 'titanium_prep.win64.exe';
-					titaniumPrepHook(
-						path.join(this.platformPath, titaniumPrep),
-						args,
-						opts,
-						function (err) {
-							if (err) {
-								fatal(err);
-							}
-							next();
+				// windows 64-bit failed, try again using 32-bit
```

Comment:
this comment no longer matches reality. I believe the exact opposite is happening: we try 32-bit and then try 64-bit.